src/http/body.rs

```diff
@@ -113,6 +113,12 @@ impl Body for Empty {
     }
 }
 
+impl Body for Stdin {
+    fn len(&self) -> Option<usize> {
+        None
+    }
+}
+
```

Comment:
Rather than implement this special case, could we export a struct BodyStream<S: AsyncRead> that gives an impl Body with len always None?

Reply:
I've now made this change.